src/client/src/StreamState.c

```diff
@@ -39,6 +39,27 @@ StateMachineState STREAM_STATE_MACHINE_STATES[] = {
 
 UINT32 STREAM_STATE_MACHINE_STATE_COUNT = SIZEOF(STREAM_STATE_MACHINE_STATES) / SIZEOF(StateMachineState);
 
+///////////////////////////////////////////////////////////////////////////
+// State machine transition iterator
+///////////////////////////////////////////////////////////////////////////
+
+STATUS iterateStreamStateMachine(PKinesisVideoStream pKinesisVideoStream)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PStateMachine pStateMachine = pKinesisVideoStream->base.pStateMachine;
```

Comment:
Add null checks for all pointers used in this function.

Reply:
Added!